app-gateway.tf

```diff
@@ -18,6 +18,9 @@ module "appGw" {
   wafName           = "${var.product}-appGW"
   resourcegroupname = "${azurerm_resource_group.rg.name}"
   use_authentication_cert = "true"
+  firewall_mode = "Detection"
+  file_upload_limit_mb = "50"
```

Comment:
can you explain the rational for the values of `file_upload_limit_mb` and `max_request_body_size_kb` please?

Reply:
`file_upload_limit_mb` is for limiting the maximum file size for document uploads, i.e. the maximum allowed file size for our document management
`max_request_body_size_kb` is for the maximum body size that is allowed for posting e.g. the size of the json data posted

Reply:
File upload limit is set to the max that dm store allows. max request body size is set to 128kb which is the max for an AppGW

Reply:
Is it really needed to enforce the max size on the AG with `file_upload_limit_mb`? Can't we just leave the dm store to control this?
I'm saying this because if we change the max size on DM store, then we will have to remember to change it here as well! And we will for sure forget
Is a kind of duplication.

Reply:
@cakeben can we please increase the file upload limit to 100MB? Business has asked to allow files up to 100MB